tools/stress-cluster/chaos/examples/network-stress-example/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.14
+FROM mcr.microsoft.com/dotnet/sdk:6.0.202-alpine3.14-amd64 
```

Comment:
Testing in local. docker build passed, but not able to verify docker run. @benbp Could you suggest how to run the dockerfile?

Reply:
@ckairen has a fix on this: https://github.com/Azure/azure-sdk-tools/pull/3240
Will remove this one.